modules/wsl-distro.nix

```diff
@@ -69,7 +70,18 @@ in
     # WSL does not support virtual consoles
     console.enable = false;
 
-    hardware.opengl.enable = true; # Enable GPU acceleration
+    hardware.opengl = {
+      enable = true; # Enable GPU acceleration
+      
+      package = mkIf cfg.useWslLib (pkgs.runCommand "wsl-lib" {} ''
+        mkdir "$out"
+        ln -s /usr/lib/wsl/lib "$out/lib"
+      '');
+    };
+
+    nix.settings.extra-sandbox-paths = mkIf cfg.useWslLib [
+      "/usr/lib/wsl"
+    ];
```

Comment:
I think the normal nixos drivers are also not part of the sandbox, so I wouldn't set this.

Reply:
Normal nixos drivers are in a store path, therefore they can be accessed from a sandbox, while this PR lets `/run/opengl-driver/lib` be a symbolic link to `/usr/lib/wsl/lib`, which is a deadlink if we don't add it to `extra-sandbox-paths`.

Reply:
They're not. The drivers are loaded via the /run/opengl-driver path, not the store.

Reply:
Then how other drivers be available in sandboxes?

Reply:
They aren't.

Reply:
Removed